prototypes/FormatPage.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE BangPatterns     #-}
+{-# LANGUAGE InstanceSigs     #-}
```

Comment:
this is VSCode doing its thing. I can remove it, but OTOH I guess it can stay as well